bin/init_bin

```diff
@@ -0,0 +1,2 @@
+export PATH=$PATH:`pwd`
```

Comment:
It seems like even after running this script, running ```which init_bin``` results in ```init_bin not found```

Reply:
Hey Professor we do not have execute permissions for this script ``` . bin/init_bin``` in ocelot